src/tools.rs

```diff
@@ -396,6 +400,30 @@ pub async fn cache_dir() -> Result<PathBuf> {
     Ok(path)
 }
 
+async fn get_http_client(root_certificate: &Option<PathBuf>, accept_invalid_certs: bool) -> Result<reqwest::Client> {
+    let mut builder = reqwest::ClientBuilder::new();
+
+    builder = builder.danger_accept_invalid_certs(accept_invalid_certs);
+
+    if let Some(root_certs) = root_certificate {
+        let mut file = tokio::fs::File::open(root_certs).await?;
```

Comment:
Why not just use `tokio::fs::read`? -> https://docs.rs/tokio/latest/tokio/fs/fn.read.html

Reply:
There also a function to help dealing with "file not found" errors: `check_target_not_found_err`.

Reply:
> Why not just use `tokio::fs::read`? -> https://docs.rs/tokio/latest/tokio/fs/fn.read.html

Because I didn't knew that was a thing :D will adapt

Reply:
> `check_target_not_found_err`

I found this in pipelines/rust/mod.rs, is it okay to create a dependency from tools.rs to there?

Reply:
I guess you can move it to a more common location.